packages/wdio-selenium-standalone-service/selenium-standalone-service.d.ts

```diff
@@ -1,10 +1,13 @@
 import { InstallOpts, StartOpts } from "selenium-standalone";
 
-declare namespace WebdriverIO {
+export namespace WebdriverIO {
     interface Config {
         seleniumLogs?: string;
         seleniumInstallArgs?: InstallOpts;
         seleniumArgs?: StartOpts;
         skipSeleniumInstall?: boolean;
     }
 }
+// unable to export it as module "@wdio/selenium-standalone-service"
+// because selenium-standalone is not a module
```

Comment:
What do you mean by that?

Reply:
I mean that we have to do
`import { InstallOpts, StartOpts } from "selenium-standalone";`
instead of 
`/// <reference types="selenium-standalone"/>`

Reply:
@ablok please check if this solution works for you

Reply:
@xlenz Does not seem to work for me. Don't know if it matters, but we are using typescript 3.2.2
The only way I can get this working is if in `webdriverio.d.ts` I export as follows:
```typescript
declare module "webdriverio" {
    export = WebdriverIO;
}
```
and then in `selenium-standalone-service.d.ts` I put:
```typescript
import { InstallOpts, StartOpts } from "selenium-standalone";

declare module "webdriverio" {
    interface Config {
        seleniumLogs?: string;
        seleniumInstallArgs?: InstallOpts;
        seleniumArgs?: StartOpts;
        skipSeleniumInstall?: boolean;
    }
}
```
This makes all errors go away. But don't ask me why ;)

Reply:
@ablok exporting without default export throws an error for me.
However changing module name to webdriverio in elenium-standalone-service.d.ts works fine for me.

Can you please check again?

Reply:
@ablok argh, I didn't add mocha selenium service to types, I was just importing file.

Now I see your problem,

```

  node_modules/@wdio/selenium-standalone-service/selenium-standalone-service.d.ts:13:20
    13     export default WebdriverIO
```